test/kvao/ttl.suite.js

```diff
@@ -14,19 +23,19 @@ module.exports = function(dataSourceFactory, connectorCapabilities) {
     it('gets TTL when key with unexpired TTL exists - Promise API',
     function() {
       return Promise.resolve(
-          CacheItem.set('a-key', 'a-value', {ttl: 1000}))
-        .delay(1)
+          CacheItem.set('a-key', 'a-value', {ttl: INITIAL_TTL}))
+        .delay(Math.max(1, Math.floor(TTL_PRECISION / 10)))
```

Comment:
maybe move this up to a var in line 26 to describe the intent? I see you're adding some sort of delay at quick glance but don't see what the formula extracts easily.

```
var minorDelayAfterInitial = Math.max(1, Math.floor(TTL_PRECISION / 10));
...
.delay(minorDelayAfterInitial)
```

or a comment like you have above on line 11 explaining the formula.